tutorials/Application.setShortcuts.md

```diff
@@ -0,0 +1,26 @@
+Sets new application's shortcut configuration
+
+# Example
+```js
+async function createWin(config) {
```

Comment:
Misleading function name. This function creates an application, but returns its main window.